src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsServer.java

```diff
@@ -25,80 +25,83 @@
 
 package com.sun.net.httpserver;
 
-import java.net.*;
-import java.io.*;
-import java.nio.*;
-import java.security.*;
-import java.nio.channels.*;
-import java.util.*;
-import java.util.concurrent.*;
-import javax.net.ssl.*;
-import com.sun.net.httpserver.spi.*;
+import java.io.IOException;
+import java.net.BindException;
+import java.net.InetSocketAddress;
+import com.sun.net.httpserver.spi.HttpServerProvider;
 
 /**
- * This class is an extension of {@link HttpServer} which provides
- * support for HTTPS. <p>
- * A HttpsServer must have an associated {@link HttpsConfigurator} object
+ * This class is an extension of {@link HttpServer} which provides support for
+ * HTTPS.
+ *
+ * <p>A {@code HttpsServer} must have an associated {@link HttpsConfigurator} object
  * which is used to establish the SSL configuration for the SSL connections.
- * <p>
- * All other configuration is the same as for HttpServer.
+ *
+ * <p>All other configuration is the same as for {@code HttpServer}.
+ *
  * @since 1.6
  */
 
 public abstract class HttpsServer extends HttpServer {
 
     /**
      */
-    protected HttpsServer () {
+    protected HttpsServer() {
     }
 
     /**
-     * creates a HttpsServer instance which is initially not bound to any local address/port.
-     * The HttpsServer is acquired from the currently installed {@link HttpServerProvider}
-     * The server must be bound using {@link #bind(InetSocketAddress,int)} before it can be used.
-     * The server must also have a HttpsConfigurator established with {@link #setHttpsConfigurator(HttpsConfigurator)}
-     * @throws IOException
+     * Creates a {@code HttpsServer} instance which is initially not bound to any
+     * local address/port. The {@code HttpsServer} is acquired from the currently
+     * installed {@link HttpServerProvider}. The server must be bound using
+     * {@link #bind(InetSocketAddress,int)} before it can be used. The server
+     * must also have a {@code HttpsConfigurator} established with
+     * {@link #setHttpsConfigurator(HttpsConfigurator)}.
+     *
+     * @throws IOException if an I/O error occurs
```

Comment:
Third change that will need to be listed in the CSR (filling out the @throws comment)

Reply:
Change has been noted in CSR (https://bugs.openjdk.java.net/browse/JDK-8254968)